shell/components/templates/default.vue

```diff
@@ -115,35 +113,6 @@ export default {
 
   },
 
-  watch: {
```

Comment:
We now record all of the setLastVisited using the navigationGuard. This means we only have to record in one location simplifying the logic of the overall feature.